kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/config/EngineConfig.kt

```diff
@@ -0,0 +1,24 @@
+package kotlinx.fuzz.config
+
+sealed interface EngineConfig
+
+interface JazzerConfig : EngineConfig {
+    val libFuzzerRssLimitMb: Int
+    val enableLogging: Boolean
```

Comment:
I think, we should rework this option
1. It doesn't "enable logging". It repeats JazzerEngine's log into console
2. It would be usefull for any `FuzzEngine` not just for jazzer. 

So, consider moving it to global config under name like "detailedConsoleLog"

Reply:
On that note, I think we borrowed more names from Jazzer and we might want to reconsider them too, for example `instrument` seems way too unclear for me just from the name